src/operator/lrn-inl.h

```diff
@@ -31,11 +31,11 @@ struct LRNParam : public dmlc::Parameter<LRNParam> {
   uint32_t nsize;
   DMLC_DECLARE_PARAMETER(LRNParam) {
     DMLC_DECLARE_FIELD(alpha).set_default(1e-4f)
-    .describe("value of the alpha variance scaling parameter in the normalization formula");
+    .describe("Alpha variance scaling parameter in the normalization formula.");
```

Comment:
Here "alpha is used as an adjective. I think it might be more intuitive to write:

"The variance scaling parameter alpha from the normalization formula."

Where is the normalization formula? How does the user find what alpha refers to? Should we have some guidance for where this reference "normalization formula" points to? What are your thoughts??